spirv_glsl.cpp

```diff
@@ -804,6 +804,9 @@ bool CompilerGLSL::ssbo_is_std430_packing(const SPIRType &type)
 
 string CompilerGLSL::layout_for_variable(const SPIRVariable &var)
 {
+	if (is_legacy_es() || (!options.es && options.version < 330))
```

Comment:
I'm a bit unsure about the desktop check here. The 420 pack is used on desktop, so it should work in anything >= 130 with that. Would it make sense to check this only for legacy ES and < 130 desktop?